### ISSUE TITLE
♿️ Search button announced incorrectly [accessibility audit #18]

### ISSUE THREAD

**Issue by jemnery** [ministryofjustice/find-moj-data#1516]
[PR](https://github.com/ministryofjustice/find-moj-data/pull/1545)

### Describe the bug.

# Observation 

## Search button announced incorrectly

Throughout the Find MoJ Data site, the search button is programmatically labelled with `aria-label="Search button"`. As a result, screen reader users—particularly those using desktop screen readers such as NVDA and JAWS—hear the control announced as "Search button button", due to the redundancy between the visible role of the element (`<button>`) and the content of the `aria-label`.

This redundancy introduces unnecessary verbosity and may cause confusion—particularly for less experienced users of assistive technologies—by repeating the word "button" unnecessarily. While the impact may be minor for advanced users, it can reduce clarity during navigation and task completion for others. Notably, this issue does not occur on mobile platforms such as iOS (including iPad VoiceOver) and Android (TalkBack).

While this does not constitute a WCAG failure in itself, it is a usability concern that affects the clarity of navigation and task completion for some users of screen readers.

![Search button highlighted on the homepage with NVDA speech viewer activated](https://uvxxxx-testproject.s3.eu-west-2.amazonaws.com/SNAG_Program-0055.png)

### Code snippet

`<button type="submit" aria-label="Search button">`

### Recommendation 
We suggest, updating the `aria-label` to remove the redundant word "button", as the element’s role is already communicated by the screen reader. A more appropriate label would be simply:

`<button type="submit" aria-label="Search">`
